anchor/Cargo.toml

```diff
@@ -0,0 +1,15 @@
+[package]
+name = "anchor"
+version = "0.1.0"
+authors = ["Libra Association <opensource@libra.org>"]
+license = "Apache-2.0"
+edition = "2018"
+publish = false
+
+# Do not add any dependencies from Libra Core
+[dependencies]
+failure = "0.1.5"
+structopt = "0.2.15"
+
+# Create a workspace separate from Libra Core
```

Comment:
Could you add a note as to why it's important to have a space separate from Libra Core?

Reply:
Also -- what do you think about moving the anchor to something like `devtools/anchor`? Then `devtools` can be its own workspace with `anchor` as a member. That would also allow other crates (e.g. test crates) in `devtools`.